recipes/_go_select/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: _go_select
+  version: {{ go_variant_version }}
+
+build:
+  number: 0
+  string: {{ go_variant_str }}
+
+test:
+  commands:
+    - exit 0
```

Comment:
Is there a better test?

Reply:
Maybe set the test requirements for the specific variant and double check that the right one got installed?

Reply:
This is just a metapackage, like the _tflow_mutex one. There is nothing installed, it is only needed to ensure the different flavors of go don't get installed on the same environment.

Reply:
@scopatz, @beckermr, is there anything else that I should do in get go-select package? I am using the template from https://github.com/jjhelmus/selectable_pkg_variants_example, and trying to close conda-forge/go-feedstock#49.